react/components/Newsletter/index.js

```diff
@@ -69,14 +70,10 @@ class Newsletter extends Component {
   }
 
   render() {
-    const {
-      placeholder = this.props.intl.formatMessage({
-        id: 'store/newsletter.placeholder',
-      }),
-      submit = this.props.intl.formatMessage({ id: 'store/newsletter.submit' }),
-      label = this.props.intl.formatMessage({ id: 'store/newsletter.label' }),
-      hideLabel,
-    } = this.props
+    const { hideLabel, intl, submit, label, placeholder } = this.props
+    const submitText = formatIOMessage({ id: submit, intl })
+    const labelText = formatIOMessage({ id: label, intl })
+    const placeholderText = formatIOMessage({ id: placeholder, intl })
```

Comment:
Some messages translated this way and other directly on the markup. It would be nice to only use it in one way, but no need if it takes time. _We do it after VTEXDay_